### PR TITLE
320 picture upload

### PR DESCRIPTION
**Pull request by corinnaj**
Finally added image upload, so events can have pictures

### CODE REVIEW

**Review by GoesOnTangents — APPROVED** [review]

**Review by cmfcmf — CHANGES_REQUESTED** [review]
Please add the new dependencies to the Vagrantfile.
https://github.com/hpi-swt2/workshop-portal/blob/dev/Vagrantfile#L28

File: Vagrantfile [hunk 1]
```diff
@@ -26,5 +26,6 @@ Vagrant.configure("2") do |config|
   config.vm.provision "shell", inline: <<-SHELL
     apt-get update
     apt-get install -y phantomjs
+    apt-get install imagemagick
```
Comment (cmfcmf) [inline Vagrantfile]: -y is missing :)

**Review by cmfcmf — APPROVED** [review]

**Comment by Baschdl** [comment]
@corinnaj This test doesn't work on my machine.

```
Event create page should not allow uploading images that are too small
     Failure/Error: expect(page).to have_text(I18n.t "events.errors.image_too_small")
       expected to find text "Das ausgewählte Bild muss mindestens 200 x 155 Pixel groß sein" in "Toggle navigation VeranstaltungenWorkshop anfragen Einloggen Veranstaltung erstellen Achtung. In einem Feld ist ein Fehler aufgetreten. Korrigiere den Fehler, um fortzufahren Veranstaltungstyp Öffentlich Privat Entwurf Speichern Veröffentlichen Name Beschreibung Dieser Editor unterstüzt Markdown Bild translation missing: en.errors.messages.mini_magick_processing_error Lokale Datei hochladen Maximale Teilnehmerzahl Zeitspanne To × Zeitspanne hinzufügen Veranstalter Kenntnisstand Bewerbungsschluss Eigene Bewerbungs-Felder Neues Feld hinzufügen Die eigenen Felder für die Bewerbung können nach dem ersten Speichern nicht mehr geändert werden. Impressum | Hasso-Plattner-Institut © 2017 Hasso-Plattner-Institut Companion Cube <3"
     # ./spec/features/event_spec.rb:267:in `block (3 levels) in <top (required)>'
```

**Comment by Baschdl** [comment]
The new Vagrantfile resolved it

**Pull request merged**